pkg/collector/corechecks/servicediscovery/envs/envs.go

```diff
@@ -33,19 +33,8 @@ type Variables struct {
 	Vars map[string]string
```

Comment:
Taking another pass on the code - nothing is accessing `Vars` directly, and instead using `Get` or `Set`
In such a case, let's un-expose this field

Reply:
done